spec/scss_converter_spec.rb

```diff
@@ -230,4 +230,60 @@ def converter(overrides = {})
 
   end
 
+  context "importing from internal libraries" do
+    let(:internal_library) { source_dir("bower_components/jquery") }
+    let(:converter) { site.getConverterImpl(Jekyll::Converters::Scss) }
```

Comment:
This will have to be [`site.find_converter_instance`](https://github.com/jekyll/jekyll/blob/d4a7fbee28323576c88bbc4b4d689d267389dc47/lib/jekyll/site.rb#L281) if `Jekyll::VERSION >= "3.0"`